config/locales/devise_invitable.en.yml

```diff
@@ -1,7 +1,9 @@
 en:
   devise:
     failure:
-      invited: "You have a pending invitation, accept it to finish creating your account."
+      invited: >
```

Comment:
what does the > do?

Reply:
It is a YAML folded style that helps create multiline and replaced the next line with spaces

> example: >
    Several lines of text,
    with some "quotes" of various 'types'
    and also a blank line

Becomes `Several lines of text, with some "quotes" of various 'types', and also a blank line`